R/zchunk_batch_electricity_water_xml.R

```diff
@@ -34,7 +34,7 @@ module_water_batch_electricity_water.xml_DISABLED <- function(command, ...) {
               "L2233.GlobalTechSCurve_elec_cool",
               "L2233.GlobalTechShrwt_elec_cool",
               "L2233.PrimaryRenewKeyword_elec_cool",
-              "L2233.PrimaryRenewKeywordInt_elec_cool", # hey kalyn you added all of the csv before here
```

Comment:
Funny

Reply:
wait I thought I pushed commit to remove this note